torch/csrc/distributed/c10d/ParamCommsUtils.hpp

```diff
@@ -65,17 +65,55 @@ class TORCH_API ParamCommsDebugInfo
   std::vector<int64_t> outputSplitSizes_;
 };
 
-
-#define RECORD_PARAM_COMMS(rank, colName, inSize, outSize, dType, inSplitSizes, outSplitSizes) \
-  auto paramCommsInfo = std::make_shared<torch::ParamCommsDebugInfo>( \
-    rank, \
-    colName, \
-    inSize, \
-    outSize, \
-    dType, \
-    inSplitSizes, \
-    outSplitSizes); \
+#define RECORD_PARAM_COMMS(                                                    \
+    seq,                                                                       \
+    WorkNccl,                                                                  \
```

Comment:
Since this is a general comm util. It probably shouldn't be named WorkNCCL (a backend specific object) and make instead named to just `Work` instead? I have another comment to clarify what is actually getting logged here.

Reply:
Yes, makes sense, will fix.

Reply:
This supposed to be the ProcessGroup object, with the seq number can uniquely identify each collective operation.